src/editor/editor.js

```diff
@@ -149,6 +149,7 @@ export default class Editor {
 
 		return Promise.resolve()
 			.then( () => {
+				this.plugins.destroy();
```

Comment:
You made it return a promise, but you don't handle this promise. Which means there's no good test for it.